lectures/getting_started_julia/introduction_to_types.md

```diff
@@ -356,9 +355,9 @@ Let's start with a trivial example where the `struct` we build has fields named
 
 ```{code-cell} julia
 struct FooNotTyped  # immutable by default, use `mutable struct` otherwise
-    a # BAD! not typed
-    b
-    c
+    a::Any # BAD! not typed
```

Comment:
We probably don't want this here since we are trying to make the point that the `::Any` is impliled.

Can you add `annotate_untyped_fields_with_any = false` to the JuliaFormatter.toml file, remove these `::Any` and then rerun the formatter to make sure it doesn't add them again?